doc/manual/source/interaction/geopackage-import-export.rst

```diff
@@ -0,0 +1,174 @@
+GeoPackage import and export Web-API 
+====================================
+
+`GeoPackage <http://www.geopackage.org/>`_ is an open, standards-based, platform-independent, portable, self-describing, compact format for transferring geospatial information.
+
+
+The GeoPackage import-export GeoGig Web-API allows to download a repository snapshot or a subset of it as a GeoPackage file, and to upload a GeoPackage file and import its vector layers into the GeoGig repository.
```

Comment:
change ``allows to download`` to ``allows for downloading``, and change ``and to upload`` to ``and for uploading``